README.md

```diff
@@ -1 +1,138 @@
-# school-library-oop
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [school-library-oop] <a name="about-project"></a>
+
+**school-library-oop**  is a web library where users can add new students or teachers. Add new books. Save records of who borrowed a given book and when.
+
+## 🛠 Built With <a name="built-with"></a>
+
+Ruby
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Ruby</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[add_student]**
+- **[add_teacher]**
+- **[add_book]**
+
```

Comment:
- [ ] [OPITONAL] As a developer, it's important to not only write clean and well-organized code but also to document your work effectively. One key aspect of this is creating a detailed and informative README file. In particular, the Key Features section of your README presents a great opportunity to showcase your creativity and vision for the project. I would highly recommend adding a compelling and descriptive feature to this section to grab the attention of recruiters or future clients. 😊
- The features you provided are perfectly valid but the wording seems a bit technical I'd suggest maybe rephrasing them to make it sound more enticing.